gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.57.0
+version=29.57.1
```

Comment:
I don't think we're introducing any new feature in pegasus to bump minor version. pegasus features should ideally remain the same. since this is backward compatible major version bump is also not required.